spec/lib/epi_deploy/release_spec.rb

```diff
@@ -29,7 +30,7 @@ def change_branch_commit(branch, commit); end
     describe "preconditions" do
       it "can only be done on the master branch" do
```

Comment:
Need a test for the input "main" branch as well

Reply:
I've changed it to test here that it is a 'primary' branch, and then added tests to the git wrapper that both main and master meet that criteria.